pkgs/data/icons/bibata-cursors/default.nix

```diff
@@ -0,0 +1,46 @@
+{ fetchFromGitHub, gksu, inkscape, stdenv, xcursorgen }:
+
+stdenv.mkDerivation rec {
+  name = "bibata-cursors-${version}";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "KaizIqbal";
+    repo = "Bibata_Cursor";
+    rev = "v${version}";
+    sha256 = "14gvpjp4gv0m59qr8wls7xs5yjx5llldyzack5kg5cg2mzk2nsml";
+  };
+
+  shellHook = ''
```

Comment:
What's this for?

Reply:
The installer script for this package tries to run `gksu chmod +x` on a file, which isn't needed. So I override it with a blank alias.

Reply:
(Leaving aside the issue of why it's trying to elevate the privileges in the first place..)

Reply:
I think you're assuming too much knowledge of how this all works on my end. Can you propose a change and tell me why it's better, please?

Reply:
So the problematic line [is this one](https://github.com/KaizIqbal/Bibata_Cursor/blob/master/build.sh#L366) in `build.sh`
Since it is not needed (as you point out) and you are not using the installer script, I think patching that line entirely in your `postPatch` hook would be more straightforward?
Something like `substituteInPlace build.sh --replace "gksu " ""`

And then you should be able to remove `gksu` from your derivation inputs entirely

Reply:
Ah. Thanks, @c0bw3b. Changing incoming.